extension/src/GodotT5Glasses.cpp

```diff
@@ -84,7 +84,47 @@ PackedFloat64Array GodotT5Glasses::get_projection_for_eye(Glasses::Eye view, dou
     return arr;    
 }
 
+void GodotT5Glasses::on_glasses_reserved() {
+	XRServer *xr_server = XRServer::get_singleton();
+	ERR_FAIL_NULL(xr_server);
+
+    auto tracker_name = std::format("{}/head", get_id());
```

Comment:
nitpicking but to be in line with OpenXR, which I think is a good long term strategy, I'd go for `/user/{}/head`